contents/handbook/engineering/working-with-max-ai.md

```diff
@@ -1,21 +1,61 @@
 ---
-title: Working with Max AI
+title: Working with our AI platform
 sidebar: Handbook
 showTitle: true
 ---
 
-Max AI is PostHog's AI-powered platform that lets users interact with PostHog's products through a chat interface. The Max AI team is responsible for building and maintaining this AI platform.
+## Technical overview
 
-## What is Max AI?
+Almost every team either is, or needs to, build AI features. As a result, we have a platform approach so that we can reuse work when everyone contributes towards our AI features. For example, we want to avoid multiple filtering experiences. Just like how we shipped HogQL instead of everyone writing their own queries.
 
-Max AI enables users to:
+Interfaces, applications, integrations, context and existing products all work together.
 
-- Ask questions about their data in natural language
-- Generate insights and reports through conversation
-- Navigate PostHog's products using AI assistance
-- Automate common analytics tasks
+NB this stuff is all very newly being built, so the ownership is somewhat chaotic. It will get easier to have appropriate teams as we start to mature out each of these areas. Please bear with us.
 
-We want Max to work with _every_ PostHog product, so that we can at some stage make a chat (or even complete automation with approval steps as necessary) the default experience for people using PostHog, then resorting to clicking the UX as the backup option for _most_ tasks.
+### Interfaces
+
+- Max
+  - Lets users ask questions or request Max does things in a chat interface, embedded into the main product
+  - Owned by Max AI team
+- Wizard (CLI)
+  - This is one of the ways we onboard new customers - they are offered an "AI Install Wizard". It sets PostHog up in their codebase automatically.
+  - Owned by Growth team
+  - Future: this should become part of a product team, not the growth team. It should continuously update the codebase on a cron job not just on initial account creation.
+- Tasks
+  - Takes ideas from Session Summaries and Error tracking, and converts these into high quality prompts and ultimately pull requests, which get shipped agentically (via Claude CLI at the moment). Automates the actual product building.
+  - Owned by LLM Analytics team
+  - Future: this should be its own team. PostHog Rules - ways of adjusting how automated it is / the workflow. Setting up experiments/feature flags automatically. Too much stuff to list.
+
+### Applications
+
+- Session summaries
+  - Watch and summarize one or many session recordings at once to look for things like UX issues. These can then be used to do things like generate Tasks.
```

Comment:
coincidentally we were just talking about this for "what is replay in the future" discussion

and i think doubling down on summarization and having the things we need to feed into be summarized is super important

i think that just counts as an un-asked for ➕💯